libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -266,6 +279,126 @@ describe('CalendarComponent', () => {
     ).toEqual('M T W T F S S');
   });
 
+  fdescribe('active month', () => {
+    let todayDate: Date;
+
+    beforeEach(() => {
+      todayDate = new Date(2021, 0, 1);
+      spectator.setInput('todayDate', todayDate);
+    });
+
+    describe('when `minDate` is set', () => {
+      let minDate: Date;
+
+      beforeEach(() => {
+        minDate = new Date(2017, 0, 1);
```

Comment:
Consider calculating from todayDate to make it clear that it's relative to that date...

Reply:
Good point. Fixed in 59d54ef3.